package.json

```diff
@@ -27,7 +27,7 @@
 		"react": "~16.8.6",
 		"react-dom": "~16.8.6",
 		"react-flexbox-grid": "~2.1.2",
-		"react-markdown": "~4.1.0",
+		"react-markdown": "^4.1.0",
```

Comment:
Can you change this back to `~`? We should force it to remain at this sem version number so we don't run into any breaking issues.